### PR TITLE
[FEAT] Tutorials

### PR DESCRIPTION
**Pull request by spencerdezartsmith**
# Description

This PR adds some Tutorial modals which will be opened when a player interacts with key features for the first time. These modals have a loose flow but really the player may click on them in any order.

Screen Shot 2022-12-02 at 10 25 25 am

They will be triggered on the first interaction with the building/menu/game and then will not trigger again.

- Bumpkin intro on first game load
- Market intro
- Fire Pit intro
- Buildings menu intro
- Workbench intro
- Travel intro

This is the first iteration of tutorials. More will be added as we see a need.

I have also removed the `$` in front of all SFL values after a team discussion determined that it doesn't make sense.
Screen Shot 2022-12-02 at 8 42 13 am

Screen Shot 2022-12-02 at 8 42 28 am

Screen Shot 2022-12-02 at 8 42 40 am

Screen Shot 2022-12-02 at 8 50 03 am

Screen Shot 2022-12-02 at 8 42 48 am

Screen Shot 2022-12-02 at 8 50 22 am

Screen Shot 2022-12-02 at 10 24 56 am

Fixes #issue

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Locally. To test you will can go through the flow but if you want to retrigger the tutorials you will need to remove the`sunflower_isles_tutorials` key from local storage.

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/farming/bakery/components/CraftingItems.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const CraftingItems: React.FC<Props> = ({ items, onClose }) => {
     if (selected.tokenAmount) {
       setToast({
         icon: tokenStatic,
-        content: `-$${selected.tokenAmount?.mul(1)}`,
```
Comment (spencerdezartsmith) [inline src/features/farming/bakery/components/CraftingItems.tsx:65]: Just removing the `$` from all these files

File: src/features/game/expansion/components/travel/IslandList.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import stoneHaven from "assets/land/islands/stone_haven.png";
 import sunflorea from "assets/land/islands/sunflorea.png";
```
Comment (spencerdezartsmith) [inline src/features/game/expansion/components/travel/IslandList.tsx:16]: Just moved these travel components into their own directory

File: src/features/game/expansion/components/travel/IslandTravel.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
```
Comment (spencerdezartsmith) [inline src/features/game/expansion/components/travel/IslandTravel.tsx:1]: Nothing has changed here just a file move.

File: src/features/game/expansion/components/travel/IslandTravelModal.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
```
Comment (spencerdezartsmith) [inline src/features/game/expansion/components/travel/IslandTravelModal.tsx:1]: Just moved this out of `IslandTravel` and into a modal component so I could add the tutorial.

File: src/features/island/buildings/Buildings.tsx [hunk 5]
```diff
@@ -41,41 +37,7 @@ export const Buildings: React.FC = () => {
           }}
         />
       </div>
-      <Modal centered show={isOpen} onHide={() => setIsOpen(false)}>
```
Comment (spencerdezartsmith) [inline src/features/island/buildings/Buildings.tsx:44]: Just moved all this into `BuildingsModal` component to do the same as above.

**Review by HBandesh — DISMISSED** [review]
Great work!

**Review by HBandesh — APPROVED** [review]
Great work!

**Pull request merged**